src/app/clients/ActionBar/Actions.tsx

```diff
@@ -0,0 +1,261 @@
+        showSelectedParcels: false,
+        errorCondition: doesNotEqualsZero,
+        errorMessage:
+            "The day overview will show the parcels for a particular date and location. It will show not the currently selected parcel. Please unselect the parcels.",
+    },
+};
+
+interface ActionsInputComponentProps {
+    pdfType: PdfType;
+    onDateChange: (newDate: Dayjs | null) => void;
+    onDriverNameChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    onCollectionCentreChange: (event: SelectChangeEvent) => void;
+    setCollectionCentre: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const ActionsInputComponent: React.FC<ActionsInputComponentProps> = ({
+    pdfType,
+    onDateChange,
+    onDriverNameChange,
+    onCollectionCentreChange,
+    setCollectionCentre,
+}) => {
+    switch (pdfType) {
+        case "Download Driver Overview":
+            return (
+                <DriverOverviewInput
+                    onDateChange={onDateChange}
+                    onDriverNameChange={onDriverNameChange}
+                />
+            );
+        case "Download Day Overview":
+            return (
+                <DayOverviewInput
+                    onDateChange={onDateChange}
+                    onCollectionCentreChange={onCollectionCentreChange}
+                    setCollectionCentre={setCollectionCentre}
+                />
+            );
+        default:
+            <></>;
+    }
+};
+
+interface ActionsButtonProps {
+    pdfType: PdfType;
+    data: ClientsTableRow[];
+    date: Dayjs;
+    driverName: string;
+    collectionCentre: string;
+}
+
+const ActionsButton: React.FC<ActionsButtonProps> = ({
+    pdfType,
+    data,
+    date,
+    driverName,
+    collectionCentre,
+}) => {
+    switch (pdfType) {
+        case "Download Shipping Labels":
+            return <ShippingLabelsModalButton data={data} />;
+        case "Download Shopping List":
+            return <ShoppingListModalButton data={data} />;
+        case "Download Driver Overview":
+            return <DriverOverviewModalButton data={data} date={date} driverName={driverName} />;
+        case "Download Day Overview":
+            return (
+                <DayOverviewModalButton collectionCentre={collectionCentre} date={date.toDate()} />
+            );
+        default:
+            <></>;
+    }
+};
+
+interface Props {
+    selected: number[];
+    data: ClientsTableRow[];
+    actionAnchorElement: HTMLElement | null;
+    setActionAnchorElement: React.Dispatch<React.SetStateAction<HTMLElement | null>>;
+    modalError: string | null;
+    setModalError: React.Dispatch<React.SetStateAction<string | null>>;
+}
+
+const Actions: React.FC<Props> = ({
+    selected,
+    data,
+    actionAnchorElement,
+    setActionAnchorElement,
+    modalError,
+    setModalError,
+}) => {
+    const [selectedAction, setSelectedAction] = useState<PdfType | null>(null);
+    const [date, setDate] = useState(dayjs(new Date()));
```

Comment:
Does dayjs not have their own way of getting the default date?

Reply:
Searched it up. We can use just `dayjs()`. Thanks <3